scripts/scripts.js

```diff
@@ -992,8 +991,8 @@ async function loadTaxonomy() {
     });
 
     // adjust meta article:tag
-
-    const currentTags = getMetadata('article:tag', true).split(',');
+    const metaTags = getMetadata('article:tag');
+    const currentTags = metaTags ? getMetadata('article:tag').split(',') : [];
```

Comment:
This is another small bug that I found, when there is no article:tag (like /blog/ page),
```getMetadata('article:tag').split(',')``` returns array with one item that is empty string = ```['']```
which results in showing debug log ```Unknown topic in tags list:....```